python/sqlflow_submitter/tensorflow/train_estimator.py

```diff
@@ -42,7 +42,9 @@ def estimator_train_and_save(estimator, model_params, save, is_pai, FLAGS,
     else:
         model_params["model_dir"] = save
 
-    classifier = check_and_load_estimator(estimator, model_params)
+    warm_start_from = save if load_pretrained_model else None
```

Comment:
Is the below snnipet more straight forward?

```python
if load_pretrained_model:
    classifier = load_from_saved_model(....)
else:
    classifier = estimator(....)
```

Reply:
OK.